packages/create/templates/hello/start_postgres_docker.js

```diff
@@ -0,0 +1,28 @@
+const { execSync } = require('child_process');
+
+if (!process.env.PGPASSWORD) {
+  console.error("Error: PGPASSWORD is not set.");
+  process.exit(1);
+}
+
+try {
+  execSync(`docker run --rm --name=dbos-db --env=POSTGRES_PASSWORD=${process.env.PGPASSWORD} --env=PGDATA=/var/lib/postgresql/data --volume=/var/lib/postgresql/data -p 5432:5432 -d postgres:16.1`);
```

Comment:
Do you think PGPASSWORD may need escaping?  Node has a better launch library call available than execSync that will deal with this.  Also, I'd like ability to use something other than 5432 (in the host, the container stays 5432 of course).

Reply:
Done and done